packages/surveys/src/components/general/FileInput.tsx

```diff
@@ -25,6 +26,7 @@ export const FileInput = ({
   fileUrls,
   maxSizeInMB,
   allowMultipleFiles,
+  uniqueId = "",
```

Comment:
if this is changing the `htmlFor` field wouldn't it maybe more consistent to also name it `htmlFor` in the props to be consistent and not let devs wonder what the uniqueId is used for.